lib/matplotlib/backend_managers.py

```diff
@@ -254,6 +254,16 @@ def add_tool(self, name, tool, *args, **kwargs):
                                'exists, not added')
             return self._tools[name]
 
+        if name == 'cursor':
+            if tool_cls != tools.SetCursorBase:
+                _api.warn_deprecated("3.5",
+                                     message="Overriding ToolSetCursor with "
+                                     f"{tool_cls.__qualname__} was only "
+                                     "necessary to provide the .set_cursor() "
+                                     "method, which is deprecated since "
+                                     "%(since)s and will be removed "
+                                     "%(removal)s")
+
```

Comment:
I don't quite follow this. From a user perspective, under which conditions is the warning emitted, and what action should they take if they see this message?

Reply:
None, in Matplotlib. It's for third-party backends, of which I'm not sure any even implement tool manager. I'll tweak the message to say so.